README.md

```diff
@@ -48,89 +48,89 @@ RUN install-php-extensions gd xdebug
 <!-- #  EDIT THE data/supported-extensions FILE INSTEAD        # -->
 <!-- #                                                         # -->
 <!-- ########################################################### -->
-| Extension | PHP 5.5 | PHP 5.6 | PHP 7.0 | PHP 7.1 | PHP 7.2 | PHP 7.3 | PHP 7.4 |
```

Comment:
You don't need to manually update this README section: it will be updated automatically once the pull request will be merged

Reply:
aha, ok, i just ran the script.